File: Assignment_6/Assignment6_3.py
"""-----------------------------------------------------------------------------
                          Assignment6_3
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
------------------------------------------------------------------------------------------
Problem statement : Write a program to print a multiplication table of a user accepted input
----------------------------------------------------------------------------------------"""
#-------------------------------------------------------------------------------
# This function prints the table of given number 
#-------------------------------------------------------------------------------
def printTableOfNumber(num):
    for i in range(1,11):
        print(f"{num} * {i} =",num*i)
#-------------------------------------------------------------------------------
# Calls printTableOfNumber(number)
#-------------------------------------------------------------------------------
def main():
    number=int(input("Enter the number for which you want to print the table:"))
    if number > 0:
        print("------------------------------------------------------")
        print(f"Printing multiplcation table of {number}...")
        print("------------------------------------------------------")
        printTableOfNumber(number)
        print("------------------------------------------------------")

    else:
        print("Please enter number grater than 0")    

#-------------------------------------------------------------------------------
# Entry point of program
#-------------------------------------------------------------------------------
if __name__=="__main__":   
    main()
#-------------------------------------------------------------------------------

File: JavaPractice/Assignment_6/Assignment6_3.java
import java.util.Scanner;
/*-----------------------------------------------------------------------------
                          Assignment6_3
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
------------------------------------------------------------------------------------------
Problem statement : Write a program to print a multiplication table of a user accepted input
--------------------------------------------------------------------------------------------------*/
public class Assignment6_3 {

	public static void main(String[] args) {
		Scanner inputObj = new Scanner(System.in);
		System.out.print("Enter number to print the table:"); 
		int number=inputObj.nextInt();
		inputObj.close();
		

		printMultiplicationTable(number);
	}

	private static void printMultiplicationTable(int number) {
		System.out.println("------------------------------------------------------");
		System.out.println("\nPrinting multiplcation table of :"+number);
		System.out.println("------------------------------------------------------");
		for(int cnt=1;cnt<=10;cnt++) {
			
			System.out.println(number+"*"+cnt+"="+(number*cnt));
		}
	}
	
}
